build-tools/tasks/docs.js

```diff
@@ -1,10 +1,9 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 const path = require('path');
-const { paramCase } = require('change-case');
```

Comment:
Cannot remove it from the whole project, because test-utils still use it. 

But the general plan is to get rid of this dependency and make the documenter package to be the source of all component name info